Readme.md

```diff
@@ -1,3 +1,8 @@
+# parallel_tests
+
+[![Gem Version](https://badge.fury.io/rb/parallel_tests.svg)](https://rubygems.org/gems/parallel_tests)
+[![Build Status](https://travis-ci.org/grosser/parallel_tests.svg)](https://travis-ci.org/grosser/parallel_tests/builds)
```

Comment:
maybe just skip the tests if it gets too messy ... otherwise if/else jruby
might be good  to not make the other tests slower

On Wed, Aug 12, 2015 at 2:36 PM, bootstraponline <notifications@github.com>
wrote:

> In Readme.md
> <https://github.com/grosser/parallel_tests/pull/424#discussion_r36917247>:
>
> > @@ -1,3 +1,8 @@
> > +# parallel_tests
> > +
> > +[![Gem Version](https://badge.fury.io/rb/parallel_tests.svg)](https://rubygems.org/gems/parallel_tests)
> > +[![Build Status](https://travis-ci.org/grosser/parallel_tests.svg)](https://travis-ci.org/grosser/parallel_tests/builds)
>
> Sure, I didn't even see it down there. I'll remove it.
>
> It looks like the timings are still not working on jruby, I'll try larger
> values.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/parallel_tests/pull/424/files#r36917247>.
>